### ISSUE TITLE
Incomprehensible Audio generated by GL-vocoder at 169K Iterations

### ISSUE THREAD

**Issue by NewEricWang** [NVIDIA/tacotron2#51]
Hi, I have trained a model after 169K Iterations by tacotron2 with batch-size=8 on a 1080TI GPU. The training data is LJ-speech. But I get  incomprehensible Audio synthesized by GL-vocoder on this model.  The text to be synthesized is "All the wardsmen alike were more or less irresponsible."
![165k](https://user-images.githubusercontent.com/18108440/42011757-782fead4-7a84-11e8-879f-980b188061a6.png)

The synthesis waveform is incomprehensible. 
[165k.zip](https://github.com/NVIDIA/tacotron2/files/2143643/165k.zip)

**Comment by rafaelvalle** [comment]
What is your training and validation loss?

**Comment by NewEricWang** [comment]
@rafaelvalle , I am sorry I am a novice in pytorch. How can I get the training and validation loss from the  log file or check-point file? How to draw the training and  validation curves? Could you give me some guides? Thank you in advance!

**Comment by deli666** [comment]
@NewEricWang You can use tensorboard:
tensorboard --logdir=outdir/logdir

**Comment by NewEricWang** [comment]
@deli666, Thank you very much!
@rafaelvalle, My training and validation loss curves are as follows:
![validation loss](https://user-images.githubusercontent.com/18108440/42026328-6195bb64-7ab6-11e8-8d52-01c24a5aba2d.png)
![validation loss](https://user-images.githubusercontent.com/18108440/42026335-673799a2-7ab6-11e8-8d17-cbcb214529ed.png)
![alignment](https://user-images.githubusercontent.com/18108440/42026351-6d5e7918-7ab6-11e8-86e8-740bc976484b.png)
![mel_predicted](https://user-images.githubusercontent.com/18108440/42026360-7283f3aa-7ab6-11e8-870b-dd89e573a9dd.png)
![mel_taget](https://user-images.githubusercontent.com/18108440/42026369-777b50e2-7ab6-11e8-98ec-bc386e1c2239.png)

**Comment by rafaelvalle** [comment]
@NewEricWang The tensorboard plot you shared goes only up to 50k iterations and did not learn attention. Synthesis should start working when the model learns attention, i.e. the alignment plot shows a diagonal line.

**Comment by NewEricWang** [comment]
@rafaelvalle , Thank you very much! I get it. So I will wait until the alignment plot shows a diagonal line.
